[WIP] App Store Homepage Redesign

cwackerfuss opened the pull request:
This is a first iteration of a homepage redesign to get users looking at Blockstack apps right away.
Please let me know how this looks via the netlify link below, and if I should make any changes!

Thanks guys!

shea256 commented:
Awesome work @cwackerfuss! Will jump in and provide some notes on copy.

shea256 commented:
Also why is the netlify deploy check failing?

cwackerfuss commented:
Taking a look at the netlify deploy now

![giphy](https://user-images.githubusercontent.com/4405356/36451589-277a56e2-1657-11e8-8a22-4a2ca952688a.gif)

cwackerfuss commented:
![2018-02-20-15-51-localhost_3000](https://user-images.githubusercontent.com/4405356/36501974-32422e26-170e-11e8-8320-b3c82b47477f.png)
Full Page Screenshot ^

larrysalibra commented on app/js/pages/WhatIsBlockstack.js:
```diff
@@ -0,0 +1,247 @@
+                    <div className="col-md-6">
+                      <h2 className="font-weight-bold">Own your data</h2>
+                      <p>
+                        Right now, app users are "digital serfs" and applications are the "digital landlords". Users don't own their data; the app owns it. Users don't control where data gets stored; they’re only allowed to store it on the application.
```
we should use "smart" punctuation:

"digital serfs" -> “digital serfs”
don't -> don’t

larrysalibra replied:
period inside quotes & change to smart quotes

the "digital landlords". -> the “digital landlords.“

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/blockstack/blockstack.org?pullRequest=517) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/blockstack/blockstack.org?pullRequest=517) before we can accept your contribution.<br/>**1** out of **2** committers have signed the CLA.<br/><br/>:white_check_mark: cwackerfuss<br/>:x: yknl<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/blockstack/blockstack.org?pullRequest=517) it.</sub>

larrysalibra commented on app/styles/main.css:
```diff
@@ -889,13 +889,12 @@ footer a:hover, footer a:focus, footer a:active {
 .btn {
```
on touch interfaces, touching the button results in a weird state with black text and a weird outline:
![img_3260](https://user-images.githubusercontent.com/597182/36819979-f4df8928-1ceb-11e8-8c78-35ba8be54a98.png)


on an iphone-size screen, the text and content being left justified looks weird - especially the app buttons - can we try centering those?

larrysalibra commented:
@cwackerfuss @yknl Another thing - the button style changes on the homepage changed the button styles on the other pages - not sure if that was intentional if someone checked to make sure they all look acceptable with the new button styles.

yknl commented:
Last commit addressed the alignment and copy issues pointed out by @larrysalibra

cwackerfuss commented:
Quick update on this: the designs are essentially final at this point after addressing feedback, just waiting for Ryan to get back so we can get some finalized copy written and approved for both pages. I don't want us to lose momentum so any help pushing this through to the finish line is more than welcome! :slightly_smiling_face:

Invision docs:
https://invis.io/WVFVH0LH8ZQ

cwackerfuss commented:
@shea256 @yknl @larrysalibra ready for eyes on copy and any general feedback.

cwackerfuss commented:
@larrysalibra @yknl Copy is tentatively approved by @shea256 and this is ready for a code review.

@muneeb-ali and @shea256, will need your final approval before deploying. Thanks!

yknl commented:
@cwackerfuss it's looking great!

I just have a few points on the copy and design which might be subjective. Let me know your thoughts. cc @aulneau @larrysalibra @shea256 @jeffdomke 

![image](https://user-images.githubusercontent.com/29081231/37543242-5957a992-2937-11e8-9006-71085f47aa48.png)
- We should keep the Logo + Wordmark (Blockstack) in the mobile view nav bar. The emphasis here is the bolded "Own your data", which lacks context. Makes me think, is this site called "Own your data"?
- It's also not very clear here to the average person how I don't I already own my data. Maybe we should add something along the lines of don't let "big web" own your data.
- I think centering the content in the hero section works better for mobile. We're doing that in the "What is Blockstack" page and it works well. Also, consistency.

___
![image](https://user-images.githubusercontent.com/29081231/37545656-ff65e382-293f-11e8-8876-ac975c0d7bf8.png)

- It looks like the side panel, with its limited amount of content, could just be a single line that goes in or under the hero section. It will give more emphasis to the apps and make the designer cleaner. 

___
![image](https://user-images.githubusercontent.com/29081231/37543270-65e3a54e-2937-11e8-85da-5dbf90433ac9.png)
- The way the apps are aligned to the left feels a little off here. Maybe aligning them to center here could fix that too. It doesn't look that bad on a computer, but the effect is more apparent when viewed from an actual phone screen. (See below)

![image](https://user-images.githubusercontent.com/29081231/37544657-159f31f2-293c-11e8-9a43-fa26337704cd.png)

- We should have more margin between the edge of the screen and the app icons, and less margin between the app icons.
- Another idea is to put them inside containers like in the Blockstack browser app screen.

___

![image](https://user-images.githubusercontent.com/29081231/37543436-f72b04de-2937-11e8-8c0b-6092c86016da.png)
- Make the column of text in the hero section a little wider on wide screens.
- We might want to mention identity in the text, since it's an important part of Blockstack

___
![image](https://user-images.githubusercontent.com/29081231/37544332-042372fe-293b-11e8-9060-f4ac83774009.png)
Maybe we can align things a bit more here
![image](https://user-images.githubusercontent.com/29081231/37544343-100ff4d4-293b-11e8-937a-b40c204983b5.png)

cwackerfuss commented:
@shea256 please check out Ken's comments above about copy. I addressed his other points concerning style already... if its okay, I think we should launch _without_ the sidebar on the homepage. Right now its not helpful and I think we should think a little deeper about how to convey the messaging.
